DungeonGame.py

```diff
@@ -18,17 +18,20 @@
 die = ("die", "Die", "I die", "I Die", "i die", "i Die", "dead", "Dead")
 inventory = ("Inventory", "inventory", "I", "i")
 quit = ("Quit", "quit", "Q", "q", "Exit", "exit", "E", "e")
+game_state = {}
```

Comment:
oh yeah I wasn't sure how to address that yet but I knew I needed it to exist outside of new_game.